test/topology_custom/test_tablets.py

```diff
@@ -301,3 +301,51 @@ async def test_read_of_pending_replica_during_migration(manager: ManagerClient,
 
     rows = await cql.run_async("SELECT pk from test.test")
     assert len(list(rows)) == 1
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_alter_dropped_tablets_keyspace(manager: ManagerClient) -> None:
```

Comment:
Let's put this in separate file so the test case can run concurrently with other test cases in test_tablets.py.

Also, does this test fail without the fix?

Reply:
Yes, the test fails without the fix.

Reply:
What's the reason for moving this specific testcase to a separate file? You want to test if ALTER can indeed run concurrently with other operations? Testing from this angle sounds OK, but this approach may not be the best, understanding why a testcase failed would be hard.

Reply:
> What's the reason for moving this specific testcase to a separate file?

Speed of the test suite i.e. how fast test suite can finish (the more test cases can run concurrently, the faster the whole suite will finish)

> You want to test if ALTER can indeed run concurrently with other operations?

No, each test case has exclusive access to a leased cluster...

Reply:
Clear, thanks, done.